src/main/java/codes/biscuit/skyblockaddons/gui/SettingsGui.java

```diff
@@ -7,13 +7,15 @@
 import codes.biscuit.skyblockaddons.utils.Feature;
 import codes.biscuit.skyblockaddons.utils.Language;
 import codes.biscuit.skyblockaddons.utils.Message;
+import codes.biscuit.skyblockaddons.utils.discord.DiscordStatus;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.gui.GuiButton;
 import net.minecraft.client.gui.GuiScreen;
 import net.minecraft.client.gui.ScaledResolution;
 import net.minecraft.client.renderer.GlStateManager;
 import net.minecraft.util.ResourceLocation;
 import net.minecraftforge.client.GuiIngameForge;
+import scala.actors.threadpool.Arrays;
```

Comment:
Don't use Arrays from scala.

Reply:
Oops, definitely not intentional, thanks!